Parallel-Tests.2017.ps1

```diff
@@ -0,0 +1,66 @@
+param(
```

Comment:
Nope, but to not to have common files with current master build/test scripts having a "copy" makes easier the last cleanup.
In the vNext folder we followed the same pattern. The file is no different.